### PR TITLE
Bump to latest version of circe and grackle

### PR DESCRIPTION
**Pull request by rpiaggio**
Both required some changes in the code, especially grackle.

### CODE REVIEW

File: gen/rules/src/main/scala/clue/gen/GraphQLGen.scala [hunk 1]
```diff
@@ -129,7 +133,8 @@ class GraphQLGen(config: GraphQLGenConfig)
                             s"Warning parsing document: ${queryResult.toProblems.map(_.toString).toList.mkString("\n")}"
                           )
                         ) >> IO {
-                          val operation = queryResult.toOption.get
+                          // TODO Support multi-operation queries?
```
Comment (cquiroz) [inline gen/rules/src/main/scala/clue/gen/GraphQLGen.scala:136]: What is a multi-operation query?

File: gen/rules/src/main/scala/clue/gen/QueryGen.scala [hunk 2]
```diff
@@ -372,11 +377,11 @@ trait QueryGen extends Generator {
...
           }
-
-        case Empty => ClassAccumulator()
-        case _     =>
-          log(s"Unhandled Algebra: [$algebra]").unsafeRunSync()
-          ClassAccumulator()
+        case Empty                                     => ClassAccumulator()
+        case _                                         =>
+          throw new Exception(
+            s"Unhandled Algebra: [$currentAlgebra] - Current Type: [$currentType]"
```
Comment (cquiroz) [inline gen/rules/src/main/scala/clue/gen/QueryGen.scala:383]: Cool exception

**Review by cquiroz — APPROVED** [review]
LGTM

File: gen/rules/src/main/scala/clue/gen/GraphQLGen.scala (hunk 1 above)
Comment (rpiaggio) [reply gen/rules/src/main/scala/clue/gen/GraphQLGen.scala:136]: It's sending multiple queries in the same request.

Here's an example: https://dgraph.io/docs/v21.03/graphql/api/multiples/

**Pull request merged**